app/assets/javascripts/initializers/initializeBaseTracking.js.erb

```diff
@@ -67,14 +66,6 @@ function eventListening(){
   }
 }
 
-function logImpressions() {
```

Comment:
This probably needs to go back in for logged out users since as discussed, logged out users will be using server-side rendering.